src/taipy/gui/viselements.json

```diff
@@ -970,7 +970,7 @@
           },
           {
             "name": "editable[<i>column_name</i>]",
-            "type": "bool",
+            "type": "dynamic(bool)",
```

Comment:
I reread the code and this is not dynamic :-(
I think the idea was that you define which columns can be edited and they become editable when the global (dynamic) editable is set to true

Reply:
If you say so. One of these days I will be able to do what you just did.